### ISSUE TITLE
modals in Dashboard aren't accessible

### ISSUE THREAD

**Issue by hughrun** [hughrun/empocketer#3]
Need to add focus to modals, enable keyboard navigation, and possibly change from `visibility: none` to make them not visible a different way.

**Comment by hughrun** [comment]
focus for all modals added in https://github.com/hughrun/empocketer/commit/b22b330ba8e71b637cd744ab298c18ec87edd031
